cmd/oci-image-tool/man/oci-image-tool-unpack.1.md

```diff
@@ -9,7 +9,7 @@ oci-image-tool-unpack \- Unpack an image or image source layout
 
 # DESCRIPTION
 `oci-image-tool unpack` unpacks a given OCI image into a directory suitable to be used with `runc`.
-
+It validates an application/vnd.oci.image.manifest.v1+json and unpacks its layered filesystem to `dest`.
```

Comment:
I think we can remove the old line here too, and just use:

   `oci-image-tool unpack` validates an application/vnd.oci.image.manifest.v1+json and unpacks its layered filesystem to `dest`.

Also, you're using `DEST` for `oci-image-tool-create-runtime-bundle` and `dest`  here.  I have a slight preference for `DEST` (caps for placeholders), but whatever we do we should be consistent ;).